src/main/java/com/cloudogu/scm/review/pullrequest/api/MergeResource.java

```diff
@@ -270,7 +270,14 @@ public MergeStrategyInfoDto getMergeStrategyInfo(
       service.isCommitMessageDisabled(strategy),
       commitDefaults.getCommitMessage(),
       service.createMergeCommitMessageHint(strategy),
-      format("%s <%s>", commitAuthor.getDisplayName(), commitAuthor.getMail())
+      renderCommitAuthorIfPresent(commitAuthor)
     );
   }
+
+  private String renderCommitAuthorIfPresent(DisplayUser commitAuthor) {
+    if (commitAuthor == null) {
+      return null;
+    }
+    return commitAuthor.getMail() == null ? commitAuthor.getDisplayName() : format("%s <%s>", commitAuthor.getDisplayName(), commitAuthor.getMail());
```

Comment:
Use `StringUtils.isNotEmpty` (empty check in addition to null check) to verify that the email is there